balearesg/bg-docs
# 😎 Password Reset

# Password Reset

## Description
This Pull Request introduces a new functionality that allows users to change their passwords in case they have forgotten them. This improves security and user experience by providing a way to regain access to their accounts autonomously.

## Changes Made
- Implementation of password change functionality in the authentication module.
- Added a "Forgot Password" link on the home screen

## How to Test New Functionality
1. Go to the login screen.
2. Click on "Forgot Password".
3. Enter the email associated with the account.
4. Check the mail for the reset link.
5. Follow the instructions to set a new password.
